server/src/__tests__/config.test.ts

```diff
@@ -1,5 +1,29 @@
 import * as config from '../config'
 
+describe('getShellcheckPath', () => {
+  it('defaults to shellcheck without path', () => {
+    process.env = {}
+    const result = config.getShellcheckPath()
+    expect(result).toEqual('shellcheck')
+  })
+
+  it('default to null in case of an empty string', () => {
+    process.env = {
+      SHELLCHECK_PATH: '',
```

Comment:
We need to add this to `vscode-client/src/extension.ts` for it to be supported by the vscode client.